lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -256,7 +256,9 @@ void SessionState::insert_pdr(
     pdr_list_.push_back(*rule);
   }
   // update criteria to be updated
-  if (crit_add) session_uc->pdrs_to_install.push_back(*rule);
+  if (crit_add) {
```

Comment:
this might be unrelated to this change, but is `crit_add` a boolean value to indicate whether to update `session_uc` or not? In that case, I think we should just either pass a nullptr or a pointer instead of passing around a boolean.

Reply:
Yes This change was un related, for readability I have added braces here, anyway I have eliminated this bool crit_add parameter and using session_uc as suggested.